public/app/features/dashboard/dashgrid/DashboardPanel.tsx

```diff
@@ -89,20 +104,35 @@ export class DashboardPanel extends PureComponent<Props, State> {
     }
   }
 
+  get screenDimensions(): DimensionsXY {
+    return {
+      x: document.documentElement.scrollWidth,
+      y: Math.floor(document.documentElement.scrollHeight * 0.4),
+    };
+  }
+
+  updatePanelDimensions = () => {
+    this.setState({
+      panelDimensions: this.screenDimensions,
+    });
+  };
+
   componentDidMount() {
     this.loadPlugin(this.props.panel.type);
+    window.addEventListener('resize', this.debouncedUpdatePanelDimensions);
   }
 
   componentDidUpdate() {
-    if (!this.element || this.state.angularPanel) {
+    if (!this.element || this.element.innerHTML !== '') {
```

Comment:
The ResizableBox component empties the this.element when resizing so if its an angular-panel we need to destroy the previous angular component and attach a new one.